routing/missioncontrol.go

```diff
@@ -93,6 +93,10 @@ type NodeResults map[route.Vertex]TimedPairResult
 // since the last failure is used to estimate a success probability that is fed
 // into the path finding process for subsequent payment attempts.
 type MissionControl struct {
+	// UpdateEstimator is a function that is called whenever the
+	// mission control state is updated.
+	UpdateEstimator func(estimator Estimator)
```

Comment:
This commit needs to come first, otherwise things won't compile (the commit that's currently the first one already references this new field but it's not yet available).

Reply:
Thank you! Done!